ClearKeep/Scenes/ChatGroup/Views/CreateGroupView.swift

```diff
@@ -0,0 +1,150 @@
+//
+//  SwiftUIView.swift
+//  ClearKeep
+//
+//  Created by đông on 09/05/2022.
+//
+
+import SwiftUI
+import Combine
+import Common
+import CommonUI
+
+private enum Constants {
+	static let spacerTopView = 90.0
+	static let spacer = 25.0
+	static let paddingVertical = 14.0
+	static let heightButton = 40.0
+	static let cornerRadiusButtonNext = 40.0
+	static let paddingButtonNext = 60.0
+	static let sizeImage = 64.0
+	static let buttonSize = CGSize(width: 196.0, height: 40.0)
+}
+
+struct CreateGroupView: View {
+	// MARK: - Variables
+	@Environment(\.presentationMode) var presentationMode
+	@Environment(\.injected) private var injected: DIContainer
+	let inspection = ViewInspector<Self>()
+	@Environment(\.colorScheme) var colorScheme
+	
+	@State private var nameGroup: String = ""
+	@State private var idClient: String = ""
+	@State private(set) var nameGroupStyle: TextInputStyle = .default
+	@Binding var loadable: Loadable<ICreatGroupViewModels>
+	@Binding var getProfile: CreatGroupProfieViewModel?
+	@Binding var clientInGroup: [CreatGroupGetUsersViewModel]
+	@State private var messageAlert: String = ""
+	@State private var showAlertPopup: Bool = false
+	@State private var showingAlert = false
+	
+	// MARK: - Body
+	var body: some View {
+		VStack {
+			VStack(alignment: .leading) {
+				Text("GroupChat.Group.Name".localized)
+					.font(AppTheme.shared.fontSet.font(style: .body2))
+					.foregroundColor(foregroundButtonBack)
+				
+				CommonTextField(text: $nameGroup,
+								inputStyle: $nameGroupStyle,
+								inputIcon: Image(""),
+								placeHolder: "GroupChat.Named".localized,
+								keyboardType: .default,
+								onEditingChanged: { isEditing in
+					nameGroupStyle = isEditing ? .normal : .highlighted
+				})
+				Text("GroupChat.User.Inside".localized)
+					.foregroundColor(foregroundText)
+					.font(AppTheme.shared.fontSet.font(style: .input2))
+			}
+			ScrollView {
+				ForEach(clientInGroup) { item in
+					ListUser(imageUrl: .constant(""), name: .constant(item.displayName))
+				}
+			}
+			RoundedGradientButton("GroupChat.Create".localized,
+								  disabled: .constant(nameGroup.trimmingCharacters(in: .whitespacesAndNewlines).isEmpty),
+								  action: creatGroup)
+				.frame(width: Constants.buttonSize.width)
+				.padding(.bottom, Constants.paddingButtonNext)
+		}
+		.padding(.horizontal, Constants.paddingVertical)
+		.onReceive(inspection.notice) { inspection.visit(self, $0) }
+		.edgesIgnoringSafeArea(.all)
+		.hiddenNavigationBarStyle()
+		.applyNavigationBarPlainStyle(title: "GroupChat.Back.Button".localized,
+									  titleColor: titleColor,
+									  backgroundColors: backgroundButtonBack,
+									  leftBarItems: {
+			BackButtonStandard(customBack)
+		},
+									  rightBarItems: {
+			Spacer()
+		})
+        .alert("GroupChat.Warning".localized, isPresented: $showAlertPopup) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/fe9f474e9dacdb67166a3b18715e917932515a0f/ClearKeep/Scenes/ChatGroup/Views/CreateGroupView.swift#L85"></span>Please use tabs instead of spaces<br />
<code>not_spaces</code> <code>CreateGroupView.swift:85</code></td>
    </tr>
  </tbody>
</table>